spec/core_functions/color/lab/error.hrx

```diff
@@ -0,0 +1,259 @@
+<===> unit/lightness/input.scss
+a {b: lab(1px 2 3)}
+
+<===> unit/lightness/error
+Error: $lightness: Expected 1px to have no units or "%".
```

Comment:
Nit: unrelated to the change, but when I read "expected 1px have no units or %" It makes me think that the number shouldn't have units and that it also shouldn't have percent (some users might not think of percent as a unit).

So I wonder if that could be reworded (maybe on a separate PR?) to something like.. `Expected 1px to have no units or to have "%".` or `Expected 1px to have "%" or no units.` or something like that 🤔